case_mapping/uco/observable.py

```diff
@@ -1385,11 +1386,12 @@ def __init__(
         self["@type"] = "uco-observable:WirelessNetworkConnectionFacet"
         self._str_vars(
             **{
-                "uco-observable:ssid": ssid,
-                "uco-observable:baseStation": base_station,
+                "uco-observable:baseStation": wn_base_station,
+                "uco-observable:password": wn_password,
+                "uco-observable:ssid": wn_ssid,
+                "observable:wirelessNetworkSecurityMode": wn_wireless_network_security_mode,
             }
         )
-        self._node_reference_vars(**{"uco-observable:location": location})
```

Comment:
Yes, this was correct to remove.  A `Relationship` needs to be used to tie a location to a base station.  Further, the connection is not the base station.